### ISSUE TITLE
Posts not loading

### ISSUE THREAD

**Issue by Maximapple** [rvaidun/befake#208]
Failed to load resource: the server responded with a status of 401 (Unauthorized)
http://localhost:8080/https://mobile.bereal.com/api/feeds/friends-v1

Missing required request header. Must specify one of: origin,x-requested-with

If anyone knows a fix, please let me know! Thanks.

**Comment by retoheusser** [comment]
Most probably the issue is that BeReal blocks the certain device id that the happy-headers dependency is using. The npm package has already been updated to use a new device id, so it should be fixable similarly to https://github.com/s-alad/toofake/pull/129

**Comment by Maximapple** [comment]
> Most probably the issue is that BeReal blocks the certain device id that the happy-headers dependency is using. The npm package has already been updated to use a new device id, so it should be fixable similarly to [s-alad/toofake#129](https://github.com/s-alad/toofake/pull/129)

Yep, that was it! Changed the happyheaders.js file within the src folder of befake and now ir works again.

Thank you! :)

**Comment by Maximapple** [comment]
He

**Comment by Maximapple** [comment]
Unfortunately, the issue is back and posts are no longer loading.

@retoheusser do you think the same fix can resolve this issue?

Thanks.

**Comment by clam0r** [comment]
Seems like now they've changed their api endpoints 

```
httpx.HTTPStatusError: Client error '404 Not Found' for url 'https://mobile.bereal.com/api/feeds/friends-v1'
For more information check: https://httpstatuses.com/404
```

**Comment by Maximapple** [comment]
> Seems like now they've changed their api endpoints
> 
> ```
> httpx.HTTPStatusError: Client error '404 Not Found' for url 'https://mobile.bereal.com/api/feeds/friends-v1'
> For more information check: https://httpstatuses.com/404
> ```

This might be fixable if someone can get the new url... right? 😅

**Comment by clam0r** [comment]
The new endpoint is `https://mobile-l7.bereal.com/api`

Dont know what else has changed, but it seems to be working

**Comment by Maximapple** [comment]
> The new endpoint is `https://mobile-l7.bereal.com/api`
> 
> Dont know what else has changed, but it seems to be working

Thank you, works again!

**Comment by Maximapple** [comment]
Hey hey, is there any chance they've changed something again? I see the loading animation when opening the BeFake site but no posts are shown and no error message appears.

![Image](https://github.com/user-attachments/assets/87e2b93c-abd2-4085-b3c2-5fba9bd468e0)
![Image](https://github.com/user-attachments/assets/c8da15c9-ee60-44cd-9199-025d2246f81a)

**Comment by clam0r** [comment]
This seems to be a BeReal bug. Realmojis from users you are not friends with will not load correctly. They are missing the user attributes and result in the above error message. 
On the BeReal app itself, the affected Realmojis are completely missing for me too.

**Comment by Maximapple** [comment]
> This seems to be a BeReal bug. Realmojis from users you are not friends with will not load correctly. They are missing the user attributes and result in the above error message. On the BeReal app itself, the affected Realmojis are completely missing for me too.

Oh, wow. I noticed this too now that you mention it.

Let's hope this is just a bug and will be fixed soon.

**Comment by Maximapple** [comment]
Posts are failing to load again. Not sure if it's just a temporary issue, though...

Failed to load resource: the server responded with a status of 500 (Internal Server Error)

[.....]/https://mobile-l7.bereal.com/api/feeds/friends-v1

Any help is appreciated!
